.github/workflows/pr-comment-terraspace.yaml

```diff
@@ -0,0 +1,189 @@
+        working-directory: terraspace/infra-prj-a
+        run: bundle install
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Terraspace - Plan
+        id: ts_plan
+        if: fromJSON(needs.pre-reqs.outputs.result).requestPlan == 'true'
+        working-directory: terraspace/infra-prj-a
+        continue-on-error: true
+        env:
+          STACK: ${{ fromJSON(needs.pre-reqs.outputs.result).subCmd }}
+        run: |
+          echo "STACK=${STACK}"
+          terraspace plan "${STACK}" --auto
+
+      - name: Terraspace - All Plan
+        id: ts_all_plan
+        if: fromJSON(needs.pre-reqs.outputs.result).requestAllPlan == 'true'
+        working-directory: terraspace/infra-prj-a
+        continue-on-error: true
+        run: terraspace all plan --out ":MOD_NAME.plan"
+
+      - name: Terraspace - All Plan Logs
+        if: steps.ts_all_plan.conclusion != 'skipped'
+        working-directory: terraspace/infra-prj-a
+        run: terraspace logs plan -a --no-timestamps
+
+      - name: Get Approval Status
+        id: get_approval_state
+        env:
+          pr_number: ${{ fromJSON(needs.pre-reqs.outputs.result).prNumber }}
+        run: |
+          res=$(curl "https://api.github.com/repos/${{ github.repository }}/pulls/${pr_number}/reviews?per_page=100" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -H 'Content-Type: application/json')
+          echo "Res: $res"
+          echo "::set-output name=approval_state::$(echo "$res" | jq '.[] | select(.state=="APPROVED").state' | sed 's/"//g')"
+
+      - name: Display Approval Result
+        env:
+          APPROVAL_STATE: ${{ steps.get_approval_state.outputs.approval_state }}
+        run: echo "approval_state=${APPROVAL_STATE}"
+
+      - name: Terraspace - Up
+        id: ts_up
+        if: |
+          fromJSON(needs.pre-reqs.outputs.result).requestUp == 'true' &&
+          steps.get_approval_state.outputs.approval_state  == 'APPROVED'
+        working-directory: terraspace/infra-prj-a
+        continue-on-error: true
+        env:
+          STACK: ${{ fromJSON(needs.pre-reqs.outputs.result).subCmd }}
+        run: terraspace up "${STACK}" --auto --yes
+
+      - name: Terraspace - All Up
+        id: ts_all_up
+        if: |
+          fromJSON(needs.pre-reqs.outputs.result).requestAllUp == 'true' &&
+          steps.get_approval_state.outputs.approval_state  == 'APPROVED'
+        working-directory: terraspace/infra-prj-a
+        continue-on-error: true
+        run: terraspace all up --yes --plan ":MOD_NAME.plan"
+
+      - name: Terraspace - All Up Logs
+        if: steps.ts_all_up.conclusion != 'skipped'
+        working-directory: terraspace/infra-prj-a
+        run: terraspace logs up -a --no-timestamps
```

Comment:
Curiosity: Are we currently using these (and plan) logs anywhere? 
Is this thinking about improving the checks section in the future?

Reply:
> Are we currently using these (and plan) logs anywhere?

If I understand the question properly, we are using it to display in the console output what the detailed results of the `plan` and `up` were for each stack